ci/scripts/install_coiled_runtime.sh

```diff
@@ -11,8 +11,13 @@ then
   mamba env update --file latest.yaml
 else
   mamba install -c conda-forge coiled-runtime=$COILED_RUNTIME_VERSION
+  # TODO: Remove before merging
```

Comment:
The latest soft env runs, the 003 and 004 runtimes don't. I thought by adding this mamba install here but it seems that doesn't do it.

Reply:
We'll want to just skip `test_xgboost_distributed_training` on the 0.0.3 and 0.0.4 builds. We should be able to move the `mamba install` below and things should pass

Reply:
I see, when you mean skip would be something like this? 
```python
import pytest
from packaging.version import Version
import coiled_runtime

@pytest.mark.skipif(
    Version(coiled_runtime.__version__) < Version("0.1.0"),
    reason="no xgboost or dask-ml packages in runtime 0.0.3 and 0.0.4",
)
def test_xgboost_distributed_training(small_client):
```

I noticed that `coiled-runtime` is in the package list but I can't import it. I thought of using this from the context but not sure what's the best practice here. 
https://github.com/coiled/coiled-runtime/blob/5f71950fed48d4de8e200e2af94610349b1e1cea/conftest.py#L66-L80